src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -67,23 +67,25 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     // the default selection handler of the entity, which the field reference
     // to, and add another logic to the query object i.e. check if the entities
     // bundle defined as group.
-    $selection_handler = $this->getSelectionHandler();
-    $query = $selection_handler->buildEntityQuery($match, $match_operator);
-
+    
+    $query = $this->getSelectionHandler()->buildEntityQuery($match, $match_operator);
+    
```

Comment:
Happy to comply though I'm really unused to such meticulousness. Are there guidelines I can read about spacing?